src/emulator/gxm/src/gxm.cpp

```diff
@@ -42,31 +42,46 @@ static const char *parameter_name(const SceGxmProgramParameter &parameter) {
     return reinterpret_cast<const char *>(bytes + parameter.name_offset);
 }
 
+static SceGxmParameterType parameter_type(const SceGxmProgramParameter parameter) {
```

Comment:
This should probably be a reference because `SceGxmProgramParameter` is a struct.